eqc_test/src/der_enc_eqc_tests.erl

```diff
@@ -0,0 +1,8 @@
+-module(der_enc_eqc_tests).
```

Comment:
The `der_enc` test(s) aren't really hooked up to the actual implementation...

Reply:
As we discussed, I am leaving this test hooked to these scripts anyway so we know it is meant to work.